developer-docs-site/static/examples/typescript/dapp-example/src/App.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import { DappSite } from "./DappSite";
+
+function App() {
+  const aptos = (window as any).aptos;
+  const location = window.location;
+
+  const [userAddress, setUserAddress] = React.useState<string | null>(null);
+  React.useEffect(() => {
+    if (!aptos) return;
+    aptos.account().then((account: string) => {
+      setUserAddress(account);
+      if (!address) setAddress(account);
+    });
+  }, [aptos]);
+
+  const [address, setAddress] = React.useState<string | null>(null);
+  React.useEffect(() => {
+    setAddress(location.pathname.slice(1).toLowerCase());
+  }, [location.pathname]);
+
+  return (
+    <div className="App">
+      {address != null ? (
```

Comment:
maybe just `{!!address ? ` or if we won't have empty string by mistake just `{address ? ...`